lms/djangoapps/teams/static/teams/js/spec/views/topic_teams_spec.js

```diff
@@ -1,21 +1,21 @@
 define([
     'backbone',
-    'teams/js/collections/team',
-    'teams/js/collections/team_membership',
     'teams/js/views/topic_teams',
     'teams/js/spec_helpers/team_spec_helpers',
-    'common/js/spec_helpers/ajax_helpers',
     'common/js/spec_helpers/page_helpers'
-], function (Backbone, TeamCollection, TeamMembershipCollection, TopicTeamsView, TeamSpecHelpers,
-             AjaxHelpers, PageHelpers) {
+], function (Backbone, TopicTeamsView, TeamSpecHelpers, PageHelpers) {
     'use strict';
     describe('Topic Teams View', function () {
         var createTopicTeamsView = function(options) {
+            options = options || {};
+            var myTeamsCollection = options.myTeamsCollection || TeamSpecHelpers.createMockTeams({results: []});
             return new TopicTeamsView({
                 el: '.teams-container',
                 model: TeamSpecHelpers.createMockTopic(),
                 collection: options.teams || TeamSpecHelpers.createMockTeams(),
-                teamMemberships: options.teamMemberships || TeamSpecHelpers.createMockTeamMemberships(),
+                canUserCreateTeam: function () {
```

Comment:
It's sad to have to duplicate this method in the tests. Is there no reasonable way to create a utility method?